openid-federation-1_0.xml

```diff
@@ -599,6 +599,8 @@
 	      Entity Statements require that
 	      the <spanx style="verb">crit</spanx> (critical) claim
 	      defined in <xref target="critClaim"/> be understand and processed.
+	      Claims in the Entity Statement whose claim names are in the array
+	      that is the value of this claim MUST be understand and processed.
```

Comment:
Should not "understand" be "understood" ?

Reply:
Thanks, fixed!